daemon/kmd/lib/kmdapi/requests.go

```diff
@@ -25,13 +25,6 @@ import (
 // swagger:ignore
 type APIV1Request interface{} // we need to tell swagger to ignore due to bug (go-swagger/issues/1436)
 
-// APIV1RequestEnvelope is a common envelope that all API V1 requests must embed
-//
-// swagger:model VersionsRequest
-type APIV1RequestEnvelope struct { // we need to tell swagger to ignore due to bug (go-swagger/issues/1436)
```

Comment:
I think it could be safety removed, but this goes beyond the scope of this task. I tried to focus here on what needed in order to fix the swagger output.